Desktop/mainwindow.cpp

```diff
@@ -836,6 +836,11 @@ void MainWindow::saveKeyPressed()
 	if (_package->isModified()) _fileMenu->save();
 }
 
+void MainWindow::saveAsKeyPressed()
+{
+	if (_package->hasDataSet()) _fileMenu->saveAs();
```

Comment:
Ah ok I see why not the isModified...

However!

There is *always* a dataSet in a package, except when exiting.
So now you can press Ctrl+Shift+S on the welcome screen and save that...

I'd suggest `if(_package->isModified() && _package->isLoaded())` or something like that

Reply:
_dataSet is null when JAPS is opened without file, and the _dataSet is set to nullptr when the workspace is closed, so on the welcome the Ctrol-Shift-S does not work...
But the isLoaded is indeed cleared. (isModified should not be used, since we should be able to do saveAs even is the file is not modified).

Reply:
> _dataSet is null when JAPS is opened without file, and the _dataSet is set to nullptr when the workspace is closed, so on the welcome the Ctrol-Shift-S does not work.

Well I could get a save-as menu without having any analyses or data.
Maybe I just added and removed a distributions analysis without data or something.
But in any case, it is not a logical function to use for this. 

In fact Id also suggest making a function/property in datasetpackage `workspaceActive` or something.
It could now just be something like `workspaceActive() { return isModified() && isLoaded() }